<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogNew</class>
 <widget class="QDialog" name="DialogNew">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>420</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>341</width>
     <height>361</height>
    </rect>
   </property>
   <property name="title">
    <string>Please input the column and rows:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="9" column="2">
     <widget class="QLineEdit" name="InputXLine">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="RowLabel">
      <property name="text">
       <string>Row</string>
      </property>
     </widget>
    </item>
    <item row="10" column="2">
     <widget class="QLineEdit" name="InputYLine">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="14" column="2">
     <widget class="QLineEdit" name="WashInputXLine"/>
    </item>
    <item row="15" column="0">
     <widget class="QLabel" name="WashInputY">
      <property name="text">
       <string>wash input y</string>
      </property>
     </widget>
    </item>
    <item row="13" column="0" colspan="2">
     <widget class="QLabel" name="OutputYLabel">
      <property name="text">
       <string>output y coordinate</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0" colspan="2">
     <widget class="QLabel" name="InputXLabel">
      <property name="text">
       <string>input x coordinate</string>
      </property>
     </widget>
    </item>
    <item row="11" column="2">
     <widget class="QLineEdit" name="OutputXLine">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="11" column="0" colspan="2">
     <widget class="QLabel" name="OutputXLabel">
      <property name="text">
       <string>output x coordinate</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QLineEdit" name="RowLine">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="15" column="2">
     <widget class="QLineEdit" name="WashInputYLine"/>
    </item>
    <item row="14" column="0">
     <widget class="QLabel" name="WashInputX">
      <property name="text">
       <string>wash input x</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0" colspan="2">
     <widget class="QLabel" name="InputYLabel">
      <property name="text">
       <string>input y coordinate</string>
      </property>
     </widget>
    </item>
    <item row="13" column="2">
     <widget class="QLineEdit" name="OutputYLine">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="16" column="0">
     <widget class="QLabel" name="WashOutputX">
      <property name="text">
       <string>wash output x</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="ColumnLabel">
      <property name="text">
       <string>Column</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLineEdit" name="ColumnLine">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="17" column="0">
     <widget class="QLabel" name="WashOutputY">
      <property name="text">
       <string>wash output y</string>
      </property>
     </widget>
    </item>
    <item row="16" column="2">
     <widget class="QLineEdit" name="WashOutputXLine"/>
    </item>
    <item row="17" column="2">
     <widget class="QLineEdit" name="WashOutputYLine"/>
    </item>
   </layout>
  </widget>
  <widget class="QToolButton" name="toolButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>26</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>420</y>
     <width>138</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Read from file...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCheck">
   <property name="geometry">
    <rect>
     <x>82</x>
     <y>370</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Input Check</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogNew</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogNew</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
